.circleci/config.yml

```diff
@@ -588,6 +588,25 @@ jobs:
       - store_test_results: *store_test_results
       - store_artifacts: *artifacts_test_results
 
+  test_osx:
```

Comment:
I'm not sure we really need this. If the bytecode comparison works, then this is actually redundant.

Reply:
ok, removed.